tuna/miopen/yaml_files/sample1.yaml

```diff
@@ -33,6 +33,10 @@ load_job:
   fin_steps: false
   tag: 'someTag'
 
+export_db:
+  enabled: true
+  file_name: 'fdb_nhwc'
```

Comment:
where is this file? what is the extension?

Reply:
@alexandraBara 
Using this args:
` "./tuna/go_fish.py miopen export_db -a ${arch} -n ${num_cu} -f --session_id ${sesh1} --filename fdb_nhwc"`

Reply:
doesnt it need an extension through? what kind of file is it? .txt? .db?